api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -168,6 +169,10 @@ message ExternalProcessor {
   // :ref:`override_message_timeout <envoy_v3_api_field_service.ext_proc.v3.ProcessingResponse.override_message_timeout>`
   // If not specified, by default it is 0, which will effectively disable the ``override_message_timeout`` API.
   google.protobuf.Duration max_message_timeout = 10;
+
+  // If true, allows the processor response to allow route cache clearing.
+  // If false, route cache clearing is ignored.
+  google.protobuf.BoolValue allow_clearing_route_cache = 11;
```

Comment:
please add another comment that this is true by default. (for backward compat purposes)

Reply:
Done.